src/panels/ActionsPanel/ActionsButton.tsx

```diff
@@ -37,7 +37,15 @@ export function ActionsButton({ uri, action: _action, height }: Props) {
   return (
     <Card p={borderPadding} h={height}>
       <Group gap={0} h={'100%'}>
-        <Button onClick={handleClick} h={'100%'} p={5} flex={1} variant={'filled'}>
+        <Button
+          onClick={handleClick}
+          h={'100%'}
+          p={5}
+          flex={1}
+          variant={'filled'}
+          color={action.color ? `rgba(${action.color.join(',')})` : undefined}
```

Comment:
See comment below and in other PR about doing the conversion from [0-1] to [0-255] here instead. This is actually where it's relevant.